.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: c
+dist: trusty
```

Comment:
I suppose this `dist` doesn't have any relation to the `DISTRO`s mentioned in the matrix below?